ResearchUXFactory/SBASurveyItem+Dictionary.swift

```diff
@@ -33,6 +33,49 @@
 
 import ResearchKit
 
+func key(_ dictionaryKey: DictionaryKey) -> String {
```

Comment:
Its a convenience method to keep the name short. Otherwise, has to be `DictionaryKey.keyname.rawValue`. This is internal to the framework module.